lib/k8s/resource_client.rb

```diff
@@ -162,20 +169,22 @@ def get?
     # @param namespace [String, NilClass]
     # @return [Object] instance of resource_class
     def get(name, namespace: @namespace)
+      response_class = response_class(path('{name}', namespace: '{namespace}')) || @resource_class
```

Comment:
I don't get this `{name}` stuff, does kube handle it because i can't find anything that replaces it with the actual name or something? Why does the path method need the params then?

Reply:
Swagger json describes api paths with variable names `{name}` and `{namespace}` (see generated classes).

Reply:
Yes but does k8s-client replace the `{name}` and `{namespace}` with something real somewhere?